dev-docs/publisher-api-reference.md

```diff
@@ -1215,4 +1216,74 @@ unsubscribe(); // no longer listening
+
+It takes one argument, an `options` object (described [below](#adservers-dfp-options)) which should be used to construct the URL.
+
+It returns a (string) URL which calls DFP, letting `options.bid` (or the auction's winning bid for this adUnit, if undefined) compete alongside the rest of the demand in DFP.
+
+For more information about the options supported by the DFP API, see [the DFP API docs](https://support.google.com/dfp_premium/answer/1068325?hl=en#env).
+
+For more information about the arguments to this method, and example usage, see the sections below:
+
++ [Options](#adservers-dfp-options)
++ [Params](#adservers-dfp-params)
++ [Example](#adservers-dfp-example)
+
+<a name="adservers-dfp-options" />
+
+#### Options
+
+This object has the following properties:
+
++ `adUnit` (Object): The adUnit which this bid is supposed to help fill.  Required.
++ `bid` (Object): The bid which should be considered alongside the rest of the adserver's demand. If this isn't defined, then we'll use the winning bid for the adUnit. Optional.
++ `params` (Object): Query params which should be set on the DFP request. These will override this module's defaults whenever they conflict.  For more information, see the next section, [Params](#adservers-dfp-params).
+
+<a name="adservers-dfp-params" />
+
+#### Params
+
+This object contains the params needed to form a URL which hits the [DFP API](https://support.google.com/dfp_premium/answer/1068325?hl=en).
+
+All params (except `iu`, mentioned below) should be considered optional. This module will choose reasonable
+defaults for all of the other required params.
+
+The `params` object has the following properties:
+
++ `iu` (String): This param *must* be included, in order for us to create a valid request.
++ `description_url` (String): This field is required if you want Ad Exchange to bid on our ad unit... but otherwise optional.
++ `cust_params` (Object): If present, this will be merged with the rest of the standard Prebid targeting params (`hb_adid`, `hb_bidder`, etc).  For more information, see the [DFP documentation on `cust_params`](https://support.google.com/dfp_premium/answer/1068325?hl=en#cust_params) and the [Example](#adservers-dfp-example) below.
+
+<a name="adservers-dfp-example" />
+
+#### Example
+
+This example shows how to pass the `options` and `params` objects to the method.  For a fuller usage example, see [Show Video Ads with DFP]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
+
+```javascript
+pbjs.requestBids({
+    bidsBackHandler: function(bids) {
+        var videoUrl = pbjs.adServers.dfp.buildVideoUrl({
+            adUnit: videoAdUnit,
+            params: {
+                iu: '/19968336/prebid_cache_video_adunit'
+                cust_params: {
+                  section: "blog",
+                  anotherKey: ["value1", "value2"]
```

Comment:
Let's confirm that this is the proper notation for passing a key with multiple values

Reply:
OK!